nncf/quantization/advanced_parameters.py

```diff
@@ -147,6 +149,7 @@ class AdvancedQuantizationParameters:
     quantize_outputs: bool = False
     inplace_statistics: bool = True
     disable_bias_correction: bool = False
+    alpha: float = 0.95
```

Comment:
I think it would be better to specify parameter `smooth_quant_alpha` or even create class `AdvancedSmoothQuantizeParameters` (like `AdvancedBiasCorrectionParameters`).

сс @alexsu52, @AlexKoff88

Reply:
I agree with renaming, but I don't see benefit in creation additional class for one parameter.